jdbc/src/java/org/apache/hive/jdbc/saml/HiveJdbcBrowserClient.java

```diff
@@ -212,18 +213,19 @@ protected void openBrowserWindow() throws HiveJdbcBrowserException {
         LOG.info(
             "Desktop mode is not supported. Attempting to use OS "
                 + "commands to open the default browser");
+        String ssoUriStr = ssoUri.toString();
```

Comment:
potential for NPE given we took out the null check above?

Reply:
The client context (JdbcBrowserClientContext) does enforce non-null SSO uri at construction time and member is final.